gax/package.json

```diff
@@ -20,6 +20,7 @@
     "object-hash": "^3.0.0",
     "proto3-json-serializer": "^2.0.0",
     "retry-request": "^7.0.0",
+    "teeny-request": "^9.0.0",
```

Comment:
`retry-request`since 7.0.0 itself already has a dependency on `teeny-request`. So, we're not even introducing a new dependency here.
The documentation of `retry-request` itself now also suggests to use `teeny-request` and no longer rely on the deprecated `request`